src/test/java/uk/gov/companieshouse/overseasentitiesapi/service/PrivateDataRetrievalServiceTest.java

```diff
@@ -43,458 +48,543 @@
+                throws IOException, URIValidationException, ServiceException {
+            var objectMapper = new ObjectMapper();
 
-    @Test
-    void testServiceExceptionThrownWhenGetManagingOfficerPrivateDataThrowsApiErrorResponseException()
-        throws IOException, URIValidationException, ServiceException {
-      var objectMapper = new ObjectMapper();
+            var overseasEntityApi = new OverseasEntityDataApi();
 
-      var overseasEntityApi = new OverseasEntityDataApi();
+            //Note: If relevant models change in private-api-sdk-java then these tests may fail. Update local JSON to reflect changes in private-api-sdk-java.
+            var boDataListApi = objectMapper.readValue(PrivateBeneficialOwnersMock.jsonBeneficialOwnerString, PrivateBoDataListApi.class);
+            var managingOfficerDataApiListApi = objectMapper.readValue(jsonManagingOfficerString, ManagingOfficerListDataApi.class);
 
-      //Note: If relevant models change in private-api-sdk-java then these tests may fail. Update local JSON to reflect changes in private-api-sdk-java.
-      var boDataListApi = objectMapper.readValue(PrivateBeneficialOwnersMock.jsonBeneficialOwnerString, PrivateBoDataListApi.class );
-      var managingOfficerDataApiListApi = objectMapper.readValue(jsonManagingOfficerString, ManagingOfficerListDataApi.class );
+            when(privateOverseasEntityDataGet.execute()).thenReturn(overseasEntityDataResponse);
+            when(overseasEntityDataResponse.getData()).thenReturn(overseasEntityApi);
 
-      when(privateOverseasEntityDataGet.execute()).thenReturn(overseasEntityDataResponse);
-      when(overseasEntityDataResponse.getData()).thenReturn(overseasEntityApi);
+            when(privateBeneficialOwnerGet.execute()).thenReturn(apiBoDataListGetResponse);
+            when(apiBoDataListGetResponse.getData()).thenReturn(boDataListApi);
 
-      when(privateBeneficialOwnerGet.execute()).thenReturn(apiBoDataListGetResponse);
-      when(apiBoDataListGetResponse.getData()).thenReturn(boDataListApi);
+            when(privateManagingOfficerDataGet.execute()).thenThrow(FourHundredAndFourException);
 
-      when(privateManagingOfficerDataGet.execute()).thenThrow(FourHundredAndFourException);
+            assertNotNull(privateDataRetrievalService.getBeneficialOwnersData(COMPANY_NUMBER));
+            assertNotNull(privateDataRetrievalService.getOverseasEntityData(COMPANY_NUMBER));
+            assertEquals(privateDataRetrievalService.getManagingOfficerData(COMPANY_NUMBER), new ManagingOfficerListDataApi(Collections.emptyList()));
+        }
+    }
 
-      assertNotNull(privateDataRetrievalService.getBeneficialOwnersData(COMPANY_NUMBER));
-      assertNotNull(privateDataRetrievalService.getOverseasEntityData(COMPANY_NUMBER));
-      assertEquals(privateDataRetrievalService.getManagingOfficerData(COMPANY_NUMBER), new ManagingOfficerListDataApi(Collections.emptyList()));
+    @Nested
```

Comment:
assume above changes are just formatting to 4 space indents.

Reply:
Ah yes, apologies, i didnt realise this!